src/components/icon/_share.less

```diff
@@ -0,0 +1,18 @@
+.share-btn {
+  color: @black;
+  font-weight: 900;
+
+  &:before {
+    content: "\f1e0";
+    font-family: "Font Awesome 5 Free";
+    display: inline-block;
+    font-style: normal;
+    font-variant: normal;
+    text-rendering: auto;
+    -webkit-font-smoothing: antialiased;
+  }
+
+  &:active {
+    color: @neonBlack;
```

Comment:
This is not visible. Please try to show off active state in a better way.

Reply:
![image](https://user-images.githubusercontent.com/26204432/94993227-171a1b00-05ad-11eb-8de4-172ced4c8573.png)

Does this look good?

Reply:
How about giving it more color? Blue or something like that

Reply:
Yes, sure!

Reply:
@shahednasser So, I am thinking to go with this:
![image](https://user-images.githubusercontent.com/26204432/95008138-ba624300-0634-11eb-9ffc-1a7f54570f7d.png)

Any suggestions?

Reply:
I think the blue looks fine but can you make a new variable called "shareActive" and set it to blue? This will make it easier to change it later on if anyone needs to